Assignment_2/Assignment_2/Models/InformationDicitonary.m

```diff
@@ -0,0 +1,62 @@
+
+
+#import "InformationDicitonary.h"
+
+@implementation InformationDicitonary
+@synthesize arrayOfInformationDicitonary;
+
+-(InformationDicitonary*)  init{
```

Comment:
You have created this class for getting raw data as form of dictionary so you can create one class method which does not require any initialization at any place which will return array of dictionary object . and you can use shorthand like    NSMutableArray *arrayOfInformationDicitonary = [NSMutableArray arrayWithObjects:
                                    @{@"firstname": @"pallavi",
                                      @"lastname" :  @"srikhakollu",
                                      @"address"  : @{
                                                     @"city": @"pune",
                                                     @"state" : @"maharashtra"},
                                      @"employeeId" : @1,
                                      @"experience" : @1,
                                      @"technology" : @"java"
                                      },
                                    @{@"firstname": @"rakshith",
                                      @"lastname" :  @"nandish",
                                      @"address"  : @{
                                              @"city": @"pune",
                                              @"state" : @"maharashtra"},
                                      @"employeeId" : @2,
                                      @"experience" : @3,
                                      @"technology" : @"c++"
                                      }
                              ,nil];
dictionary structure look like this as clean.